src/integration/sync_integration_test.go

```diff
@@ -0,0 +1,89 @@
+package integration
+
+import (
+	"github.com/EaseApp/web-backend/src/sync"
+	"github.com/gorilla/websocket"
+	"github.com/stretchr/testify/assert"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+	"time"
+)
+
+func TestSocketConnection(t *testing.T) {
+	server := setUpSyncServer(t)
+
+	testcases := []struct {
+		subscribeTo  string
+		publishTo    string
+		publishData  string
+		expectedData string
+		conn         *websocket.Conn
```

Comment:
This field isn't necessary, it's just used locally in the for loop below, make it a local variable there.